layers/+completion/auto-completion/packages.el

```diff
@@ -284,6 +281,9 @@
     ;; disable yas minor mode map
     ;; use hippie-expand instead
     (setq yas-minor-mode-map (make-sparse-keymap))
+    (with-eval-after-load 'hippie-exp
+      ;; Try to expand yasnippet snippets based on prefix
+      (add-to-list 'hippie-expand-try-functions-list 'yas-hippie-try-expand))
```

Comment:
`hippie-expand-try-functions` is autoloaded (for historical reasons) so no need for `with-eval-after-load`

Also, IMO we can wait until `:config` to add this one.  It does nothing unless `yas-minor-mode` is enabled.